src/main/java/com/kobylynskyi/graphql/codegen/kotlin/KotlinGraphQLCodegen.java

```diff
@@ -49,6 +49,9 @@ protected void initDefaultValues(MappingConfig mappingConfig) {
         if (mappingConfig.isGenerateModelOpenClasses() == null) {
             mappingConfig.setGenerateModelOpenClasses(false);
         }
+        if (mappingConfig.isAutomaticallyDefaultNullable() == null) {
+            mappingConfig.setAutomaticallyDefaultNullable(true);
```

Comment:
Here you define a default value of "true", while in `codegen-options.md` and in `MappingConfigConstants.java` you have false. This can confuse library users. Can you please clarify? Thanks

Reply:
Well spotted. Will update.